styles.css

```diff
@@ -0,0 +1,71 @@
+html,
+body {
+  height: 100%;
+}
+body {
+ margin-top: 15px;
+ color: #5A5A5A;
+ text-align: center;
+ font-size: 13px;
+ font-weight: 300;
+ font-family: "Open Sans", sans-serif;;
+
+}
+
```

Comment:
I recommend being cognizant of your extra newlines here & throughout the file. The CSS style we're doing (that is agreed upon on the Interwebs) looks like this
```css
.my-class {
}
.another-class {
}
```